examples/addons/csi-secrets-store/README.md

```diff
@@ -0,0 +1,84 @@
+# CSI Secrets Store Add-on
+
+[The CSI Secrets Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) integrates secrets stores with Kubernetes via a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) volume. 
+
+With the CSI secrets store and the azure provider installed, developers can access application-specific secrets, keys, and certs stored in Azure Key Vault directly from their pods.
+
+This addon is available for 1.15+ clusters
+
+**Note** This addon is enabled by default for 1.16+ clusters
+
+To enable on 1.15 cluster, add this addon to the apimodel and also set the feature gate `CSIInlineVolume=true` for API server and kubelet config
```

Comment:
Should we remove this whole thing if we only want to support 1.16+ for secrets store csi?

Reply:
+1